jira_connector.py

```diff
@@ -514,6 +544,10 @@ def _get_custom_fields_for_issue(self, issue_id, action_result):
                 {}".format(error_message)
             return action_result.set_status(phantom.APP_ERROR, error_text), None, None
 
+        custom_fields, err = self._validate_and_update_custom_fields_url(custom_fields)
```

Comment:
I don't find return types of this method consistent. And error handling looks very golang-like. 

Maybe it'd be better to try and catch exceptions in  `_get_custom_fields_for_issue`? instead of `if err`

Reply:
resolved